sync/class.jetpack-sync-module-callables.php

```diff
@@ -138,7 +151,7 @@ public function maybe_sync_callables() {
 		foreach ( $callables as $name => $value ) {
 			$checksum = $this->get_check_sum( $value );
 			// explicitly not using Identical comparison as get_option returns a string
-			if ( ! $this->still_valid_checksum( $callable_checksums, $name, $checksum ) && ! is_null( $value ) ) {
+			if ( $this->should_send_callable( $callable_checksums, $name, $checksum ) && ! is_null( $value ) ) {
```

Comment:
since `is_null( $value )` is very cheap to execute, we should probably put it in front of the more expensive `$this->should_send_callable()` function in this if-statement, since we can shortcut execution that way.